cmd/ipfs-cluster-service/main.go

```diff
@@ -304,6 +333,22 @@ remove the %s file first and clean any Raft state.
 					checkErr("saving "+DefaultIdentityFile, err)
 					out("new identity written to %s\n", identityPath)
 				}
+				if peersOpt != "" {
+					_, ident, cfgs := makeAndLoadConfigs()
+
+					ctx := context.Background()
+					host, _, _, err := ipfscluster.NewClusterHost(ctx, ident, cfgs.clusterCfg)
```

Comment:
This is an overkill approach (creating a fully configured libp2p peer).

You can just create a peerManager with a nil host and call `SavePeerstore` with the right `AddrInfo` objects. And you have this to help you: https://godoc.org/github.com/libp2p/go-libp2p-core/peer#AddrInfosFromP2pAddrs